README.md

```diff
@@ -1,31 +1,181 @@
-# IoT Reference Integration on STM32U5 Arm Cortex-M33 MCU
+# Getting Started Guide for STM32U5 IoT Discovery Kit with AWS 
 
+## Overview 
 
-### Perfroming Over-the-air (OTA) Firmware Update for [B-U585I-IOT02A discovery kit](https://www.st.com/en/evaluation-tools/b-u585i-iot02a.html)
+The STM32U585 IoT Discovery kit comes with 2048 KB of Flash memory, 786 kB of RAM and is based on Arm® Cortex®-M33.   
```

Comment:
Suggest to add 1-2 sentences introducing software: "This project demonstrates how to integrate modular FreeRTOS software with hardware enforced security to create secure and updatable cloud connected applications. The project is pre-configured to run on the STM32U585 IoT Discovery Kit. The kit comes with ..."